### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by lloydlobo** [esquires/neorg-gtd-project-tags#3]
## What

The instructions in the README does not work when installing this plugin with packer.

## Where

[Installation](https://github.com/esquires/neorg-gtd-project-tags#installation) section in `README.md`

```lua
-- packer installation
use {'esquires/gtd-project-tags'}

...

```

## Proposal

- Simply correcting the repository name in the `use` command works!

```lua
-- packer installation
use {'esquires/neorg-gtd-project-tags'}

...

```

**Comment by esquires** [comment]
Thanks! This is now fixed in (535690b)

**Comment by lloydlobo** [comment]
You're welcome!
